gcn/notices/fermi/gbm/strict/alert.schema.json

```diff
@@ -0,0 +1,168 @@
+    },
+    "id": {
+      "type": "array",
+      "items": {
+        "oneOf": [
+          {
+            "type": "string"
+          },
+          {
+            "type": "number"
+          }
+        ]
+      },
+      "description": "Instrument-specific trigger ID, for example: 745121685, bn240812094"
+    },
+    "instrument": {
+      "const": "GBM",
+      "default": "GBM",
+      "description": "Name of the instrument reporting the event"
+    },
+    "latitude": {
+      "type": "number",
+      "description": "Geocentric latitude [deg]"
+    },
+    "light_curve_url": {
+      "type": "string",
+      "description": "URL of the quicklook light curve plot"
+    },
+    "longitude": {
+      "type": "number",
+      "description": "Geocentric longitude [deg]"
+    },
+    "messenger": {
+      "const": "EM",
+      "default": "EM",
+      "description": "Messenger of report; EM, GW or Neutrino"
+    },
+    "mission": {
+      "const": "Fermi",
+      "default": "Fermi",
+      "description": "Name of Mission or Telescope reporting the event"
+    },
+    "notice_type": {
+      "const": "FERMI_GBM_ALERT",
+      "description": "Notice title"
+    },
+    "rate_duration": {
+      "type": "number",
+      "description": "Duration in seconds of the ime interval used in the signal to noise ratio calculation"
+    },
+    "rate_energy_range": {
+      "type": "array",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 2,
+      "maxItems": 2,
+      "description": "Energy bounds in keV used in the signal to noise ratio calculation"
+    },
+    "rate_snr": {
+      "type": "number",
+      "description": "Signal to noise ratio"
+    },
+    "record_number": {
+      "type": "number",
+      "description": "Incremental number for notices related to the same trigger (ex: 1, 2, 3)"
+    },
+    "spectrum": {
+      "const": "energy",
+      "default": "energy",
+      "description": "high-energy or optical or radio observations, if not parsed, then default band is energy"
+    },
+    "trigger_algorithm_number": {
+      "type": "number",
+      "description": "The criterion number that caused the flight software to trigger. Not to be confused with flight software or any other version number."
+    },
+    "trigger_time": {
+      "type": "string",
+      "description": "Time of the trigger [ISO 8601], ex YYYY-MM-DDTHH:MM:SS.ssssssZ"
+    },
+    "trigger_type": {
+      "const": "rate",
+      "default": "rate",
+      "description": "Type of trigger algorithm used to identify the transient event."
+    },
+    "units": {
+      "const": "keV",
+      "default": "keV",
+      "description": "Units of band range, if not parsed, then default energy is keV"
```

Comment:
```suggestion
      "description": "Units of energy band range, if not parsed, then default energy is keV"
```

Reply:
@Vidushi-GitHub: Replaced "band range" with "rate_energy_range", the explicit field name. It wasn't clear. Thanks for pointing this out.